jobs/notebook-report/util/logging.py

```diff
@@ -1,16 +0,0 @@
-# setup logging - important to set it up first
-from config import Config
-from os import path
-import logging.config
-import sys
-
-
-def setup_logging(conf='logging.conf'):
-    # log_file_path = path.join(path.dirname(path.abspath(__file__)), conf)
-    log_file_path = path.join(Config.PROJECT_ROOT, conf)
-
-    if path.isfile(log_file_path):
```

Comment:
would recommend keeping this, the way you're using structured logging now doesn't support various logging levels and /or logging configuration